File: Logic_HandsOn/Java/Problems on bit,file handling,String etc/Program104.java
//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		Write a program which accept string from user and display only digits from that string.
//
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////


import java.util.Scanner;

class Solution
{
	String str;
	int iCnt;

	Solution(String str)
	{
		this.str=str;
		this.iCnt=0;
	}

	void DisplayDigit()
	{
		
		while(iCnt!=str.length())
		{
			if(str.charAt(iCnt)>=48 && str.charAt(iCnt)<=57)
			{
				System.out.print(str.charAt(iCnt));
			}
			
			iCnt++;
		}
		System.out.println();
	}
}

class Program
{
	public static void main(String args[])
	{
		Scanner sObj = new Scanner(System.in);
		System.out.println("Enter the string: ");
		String str = sObj.nextLine();

		Solution obj = new Solution(str);
		obj.DisplayDigit();
	}
}

/*
OUTPUT:

Enter the string:
Vishal89PAtil121
89121

*/

File: Logic_HandsOn/C++/Problems on bit,File Handing,Pattern etc/program104.cpp
//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		Write a program which accept string from user and display only digits from that string.
//
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////


#include<iostream>
using namespace std;

class Solution
{

	private:
		string str;
		int iCnt,iCount;

	public:

		Solution(string str,int iCnt=0,int iCount=0)
		{
			this->str=str;
		}

		void DisplayDigit()
		{
			while(iCnt!=str.length())
			{
			
				if(str[iCnt]>=48 && str[iCnt]<=57)
				{
					cout<<str[iCnt];
				}
			
				iCnt++;	
			}	
			cout<<"\n";
		}
};

int main()
{
	string str;

	cout<<"Enter string: ";
	getline(cin,str);
	
	Solution *sObj = new Solution(str);
	sObj->DisplayDigit();

	delete sObj;
	return 0;

}

/*
OUTPUT:
Enter string: marve89llous121      
89121


*/
